importer/DeHeDe.py

```diff
@@ -10,10 +10,16 @@ def set_location(self):
             location_q = utils.q_from_wikipedia("de", self.ortsteil)
             if location_q:
                 self.add_statement("location", location_q)
+            else:
+                self.add_to_report("ortsteil", self.ortsteil, "location")
 
     def set_adm_location(self):
+        city_q = None
```

Comment:
Don't think you need to set this since `utils.q_from_wikipedia` returns `None` on failiure

Reply:
Removed it.